linera-views-derive/src/lib.rs

```diff
@@ -93,15 +94,27 @@ fn generate_view_code(input: ItemStruct, root: bool) -> TokenStream2 {
             }
         });
         num_init_keys_quotes.push(quote! { #g :: NUM_INIT_KEYS });
+
+        let derive_key_logic = if num_fields < 256 {
+            let idx_byte = idx as u8;
+            quote! {
+                let index_byte = #idx_byte;
+                let base_key = context.base_key().derive_tag_key(linera_views::views::MIN_VIEW_TAG, &index_byte)?;
+            }
+        } else {
+            quote! {
+                let index = #idx_lit;
+                let base_key = context.base_key().derive_tag_key(linera_views::views::MIN_VIEW_TAG, &index)?;
```

Comment:
Can we make sure that the integer type is (locally) visible when we use `derive_tag_key` on integers? Those types are part of the protocol and any change would be breaking.